cldf/cldf-metadata.json

```diff
@@ -12,37 +12,37 @@
     "dc:related": null,
     "dc:source": "sources.bib",
     "dc:title": "Sino-Tibetan Database of Lexical Cognates",
-    "dcat:accessURL": "https://github.com/lexibank/sagartst",
+    "dcat:accessURL": "https://github.com/digling/sinotibetan-data",
     "prov:wasDerivedFrom": [
         {
             "rdf:type": "prov:Entity",
             "dc:title": "Repository",
-            "rdf:about": "https://github.com/lexibank/sagartst",
-            "dc:created": "v1.1.0-9-g15fc4f6"
+            "rdf:about": "https://github.com/digling/sinotibetan-data",
+            "dc:created": "v1.1.0-12-g1f18454"
         },
         {
             "rdf:type": "prov:Entity",
-            "dc:title": "CLTS",
-            "rdf:about": "https://github.com/cldf-clts/clts",
-            "dc:created": "v1.4"
+            "dc:title": "Glottolog",
+            "rdf:about": "https://github.com/lingulist/glottolog-data",
```

Comment:
Same here: Make sure you have the authoritative repos as `origin` remote of the repository.